ui/src/app/pages/workspace/workspace-card.spec.tsx

```diff
@@ -50,34 +50,6 @@ describe('WorkspaceCard', () => {
     serverConfigStore.set({config: {gsuiteDomain: 'abc'}});
   });
 
-  it('fetches user roles before opening the share dialog', async() => {
```

Comment:
removed b/c the functionality is tested automatically through the workspaceApi stub in workspace-share.spec